frontend/server/src/Controllers/Course.php

```diff
@@ -2340,6 +2340,55 @@ public static function getIntroDetails(\OmegaUp\Request $r): array {
         );
         $requestUserInformation = $courseDetails['requests_user_information'];
         $inContest = false;
+        $isCourseAdmin = \OmegaUp\Authorization::isCourseAdmin(
+            $r->identity,
+            $course
+        );
+        if ($isCourseAdmin && !$showAssignment) {
+            return [
+                'smartyProperties' => [
+                    'showRanking' => true,
+                    'payload' => [
+                        'details' => self::getCommonCourseDetails(
+                            $course,
+                            $r->identity,
+                            /*$onlyIntroDetails=*/ false
+                        ),
+                        'progress' => \OmegaUp\DAO\Courses::getAssignmentsProgress(
+                            $course->course_id,
+                            $r->identity->identity_id
+                        ),
+                    ],
+                ],
+                'template' => 'course.details.tpl',
+                'inContest' => $inContest,
```

Comment:
nit:

```suggestion
                'inContest' => false,
```

así se puede eliminar L2342. igual en los demás lugares.

Reply:
Listo